src/AutoBuild.jl

```diff
@@ -314,7 +314,7 @@ function get_compilers_versions(; compilers = [:c])
     return output
 end
 
-function upload_to_github_releases(repo, tag, path; gh_auth=github_auth(;allow_anonymous=false), 
+function upload_to_github_releases(repo, tag, path; gh_auth=Wizard.github_auth(;allow_anonymous=false), 
```

Comment:
Importing stuff from `Wizard` isn't exactly something I wanted to do, but for the time being it does the job